apis/dkjson

```diff
@@ -11,7 +11,7 @@ David Kolf's JSON module for Lua 5.1/5.2
 This module writes no global values, not even the module table.
 Import it using
 
-    json = require ("dkjson")
+    json = require ("github.rom/apis/dkjson")
```

Comment:
I don't think that's true, and read somewhere that require was disabled in cc.

However, you're missing my point

# this line is within a comment! It does not execute, so why does it need to change?